scale/ingest/views.py

```diff
@@ -179,8 +181,7 @@ def list_impl(self, request):
         :rtype: :class:`rest_framework.response.Response`
         :returns: the HTTP response to send back to the user
         """
-
-        started = rest_util.parse_timestamp(request, 'started', rest_util.get_relative_days(7))
+        started = rest_util.parse_timestamp(request, 'started', required=False)
```

Comment:
Are we worried about a malformed date/time string in the request?

Reply:
This is handled in the the rest_util function call.

Reply:
Looks good then.